packages/remark-lint-strikethrough-marker/index.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @author Denis Augsburger
+ * @copyright 2021 Denis Augsburger
+ * @license MIT
+ * @module strikethrough-marker
+ * @fileoverview
+ *   Warn for violating strikethrough markers.
+ *
+ *   Options: `'consistent'`, `'~'`, or `'~~'`, default: `'consistent'`.
+ *
+ *   `'consistent'` detects the first used strikethrough style and warns when
+ *   subsequent strikethrough use different styles.
+ *
+ *   ## Fix
+ *
+ *   See [Using remark to fix your Markdown](https://github.com/remarkjs/remark-lint#using-remark-to-fix-your-markdown)
+ *   on how to automatically fix warnings for this rule.
+ *
+ * @example {"setting": "~", "name": "ok.md", "gfm": true}
```

Comment:
For remark 13+, gfm moved to a package rather than an option https://github.com/remarkjs/remark/releases/tag/13.0.0, you've noted this in the readme, but this seems to refer to the old syntax?

Reply:
@ChristianMurphy nope, this is good. The tests and docs take this field and add the plugin and docs about it automatically